app/src/main/java/com/eveningoutpost/dexdrip/xdrip.java

```diff
@@ -15,5 +19,12 @@
     public void onCreate() {
         super.onCreate();
         Fabric.with(this, new Crashlytics());
+        CollectionServiceStarter collectionServiceStarter = new CollectionServiceStarter(getApplicationContext());
```

Comment:
It's a bit pedantic, but 'getApplicationContext()' is a bit redundant when inside an application 'Context'.  'this' would suffice.